NewForce-Cohort-6/tabloidmvc-candy-corn
# SB/23-View-All-User-Profiles

# Description

This PR adds the ability for users to view a list of all user profiles sorted by user display name.

NOTE: This ability needs to be restricted to ADMIN users only as instructed in Trello ticket #33.
NOTE: This branch was created from SB/29-Register branch and includes register functionality.

# Testing Instructions

1. Launch application from SB/23-View-All-User-Profiles branch
2. In the app, Login as admin@example.com
3. Go to "Users" on the nav bar
5. You should be greeted with a list of Users from the database's UserProfile table
6. Each user in the list should display the full name, the display name and the user type
7. The list should be ordered alphabetically by display name

# Checklist:
